custom/icds_reports/reports/disha.py

```diff
@@ -86,20 +92,21 @@ def _write_data_in_chunks(self, temp_path):
         with open(temp_path, 'w', encoding='utf-8') as f:
             f.write(metadata_line)
             written_count = 0
-            for _, end, total, chunk in batch_qs(indicators, chunk_size):
+            for _, end, total, chunk in batch_qs(indicators, num_batches=10):
                 chunk_string = json.dumps(list(chunk), ensure_ascii=False)
                 # chunk is list of lists, so skip enclosing brackets
                 f.write(chunk_string[1:-1])
                 written_count += len(chunk)
                 if written_count != total:
                     f.write(",")
+                logger.info("Processed {end}/{total} records".format(end=end, total=total))
```

Comment:
nit: Can instead say the batch number instead?
`Processed batch 1/10 records, total: {total}`

That would be more useful to know which batch things are at considering thats the control.

Reply:
I believe a realistic example would be
`12341212/1290000`,  which would hard for someone to just read and calculate in head

Reply:
Hmm. I don't think it will be that bad either. This requires enumerating the control and makes the code a bit more confusing. I can add it anyway